mlox_user.txt

```diff
@@ -3856,6 +3856,92 @@ TR_Mainland.esm
 Indoril_armor.esp
 Indoril_armor - TR.esp
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; @Interesting Outfits - Cultists, @IO - Cultists [TipsyTekPriest]
+
+(Ref: https://www.nexusmods.com/morrowind/mods/51922?tab=docs )
+
+[Order]
+The_Fires_of_Orc.ESP
+IO - Cultists.esp
+IO - The Fires of Orc Patch.ESP
+
+[Order]
+Daedric Shrine Overhaul Malacath.ESP
+IO - Cultists.esp
+
+[Order]
+Daedric Shrine Overhaul Mehrunes Dagon.ESP
+IO - Cultists.esp
+
+[Order]
+Daedric Shrine Overhaul Molag Bal.ESP
+IO - Cultists.esp
+
+[Order]
+Daedric Shrine Overhaul Sheogorath.ESP
+IO - Cultists.esp
+
+[Patch]
+	A patch for "Interesting Outfits - Cultists" and "The Fires of Orc" is provided on the "Interesting Outfits - Cultists" mod page.
+	[Patch Download]( https://www.nexusmods.com/morrowind/mods/51922 )
+IO - The Fires of Orc Patch.ESP
+[ALL	The_Fires_of_Orc.ESP
+		IO - Cultists.esp]
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; @Interesting Outfits - Kinsmer, @IO - Kinsmer [TipsyTekPriest]
+
+(Ref: https://www.nexusmods.com/morrowind/mods/52167?tab=docs )
+
+[Order]
+Beautiful cities of morrowind.esp
+IO - Kinsmer.ESP
+
+[Order]
+Uvirith's Legacy_3.53.esp
+IO - Kinsmer.ESP
+IO - UL Compatibility Patch.ESP
+
+[Patch]
+	A patch for "Interesting Outfits - Kinsmer" and "Uvirith's Legacy" is provided on the "Interesting Outfits - Kinsmer" mod page.
+	[Patch Download]( https://www.nexusmods.com/morrowind/mods/52167 )
+IO - The Fires of Orc Patch.ESP
```

Comment:
This says it's a patch for UL but the rule is about fires of orc

Reply:
Whoops! I've corrected this now